src/main/java/frc/robot/Robot.java

```diff
@@ -279,28 +305,47 @@ public void teleopSetup() {
     elevator.start();
     shooter.start();
     ballSubsystem.start();
-  }
 
+
+  }
+  TalonSRX turretMotor = new TalonSRX(RobotMap.TURRET_PORT);
+  TalonFX shooterMotor = new TalonFX(RobotMap.SHOOTER_PORT);
   /** This function is called periodically during operator control. */
   @Override
   public void teleopRoutine() {
-      double forward = this.driverPad.getValue(ControllerBindings.RIGHT_STICK, ControllerBindings.Y_AXIS);
-      double turn = this.driverPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.X_AXIS);
+      double forward, turn;
+
+
+      forward = -this.operatorPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.Y_AXIS);
```

Comment:
Why are the drivetrain movement values being pulled from the `operatorPad` - shouldn't this still be `driverPad`?

Reply:
at the moment one of the controllers is junk so we are just operating from one (other one stops working as soon as you move the controller basically).

Reply:
Okay, then go ahead and keep that kind of change to your local working copy of the code. I don't think it should be committed to the repo.